Tree/readme.md

```diff
@@ -47,6 +47,7 @@ that, this property should be satisfied at every node in the tree.
 * Sorting the elements of binary search tree and many more
 
 ## Questions :
+* Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp) 
```

Comment:
isn't this line already there at Line 53?

Reply:
also make sure it's in alphabetical order.

Reply:
sorry I could not understand what you are saying